src/Commands/MpxImporter.php

```diff
@@ -157,10 +157,10 @@ public function warmCache(string $media_type_id) {
   private function selectAll(MediaTypeInterface $media_type): ObjectListIterator {
     $media_source = DataObjectImporter::loadMediaSource($media_type);
     $factory = $this->dataObjectFactoryCreator->fromMediaSource($media_source);
-    $fields = new ByFields();
-    $event = new ImportSelectEvent($fields, $media_source);
+    $query = new ObjectListQuery();
+    $event = new ImportSelectEvent($query, $media_source);
     $this->eventDispatcher->dispatch(ImportSelectEvent::IMPORT_SELECT, $event);
-    $results = $factory->select($fields, $media_source->getAccount());
+    $results = $factory->select($query, $media_source->getAccount());
```

Comment:
I see you're changing here the argument passed to $factory->select, but I can't see you had changed that in the select() definition.

Reply:
Ah yes, `select()` is in the mpx-php library. This PR is addressing the API changes between the 0.3.0 and 0.4.0 releases. See https://github.com/Lullabot/mpx-php/pull/129/files